source/server/entities/teleporter.qc

```diff
@@ -295,8 +297,27 @@ void() teleport_touch =
                 people.tele_target = find(world, targetname, self.target);
                 people.movetype = MOVETYPE_NONE;
             }
-            people = people.chain;
+			people = people.chain;
         }
+		
+		entity zod = findradius(self.origin, self.stance);
```

Comment:
Please add this logic to the existing findradius() for performance concerns — you can change the `people` reference instead and just have a branch:

```c
if (tempe.classname == "player") {
    // teleport logic
} else if (tempe.aistatus == "1") {
    // ai logic (dogs, zombies)
}
```